test_wazuh/test_fim/test_recursion_level/data/wazuh_conf.yaml

```diff
@@ -0,0 +1,165 @@
+      value: "/test_recursion_320"
+      attributes:
+      - check_all: 'yes'
+      - recursion_level: "320"
+  - directories:
+      value: "/test recursion 320"
+      attributes:
+      - check_all: 'yes'
+      - recursion_level: "320"
+# conf 2
+- tags:
+  - realtime
+  apply_to_modules:
+  - test_recursion_level
+  section: syscheck
+  elements:
+  - disabled:
+      value: 'no'
+  - directories:
+      value: "/test_no_recursion"
+      attributes:
+      - check_all: 'yes'
+      - realtime: 'yes'
+      - recursion_level: "2"
+  - directories:
+      value: "/test no recursion"
+      attributes:
+      - check_all: 'yes'
+      - realtime: 'yes'
+      - recursion_level: "2"
+  - directories:
+      value: "/test_recursion_1"
+      attributes:
+      - check_all: 'yes'
+      - realtime: 'yes'
+      - recursion_level: "3"
+  - directories:
+      value: "/test recursion 1"
+      attributes:
+      - check_all: 'yes'
+      - realtime: 'yes'
+      - recursion_level: "3"
+  - directories:
+      value: "/test_recursion_5"
+      attributes:
+      - check_all: 'yes'
+      - realtime: 'yes'
+      - recursion_level: "7"
+  - directories:
+      value: "/test recursion 5"
+      attributes:
+      - check_all: 'yes'
+      - realtime: 'yes'
+      - recursion_level: "7"
+  - directories:
+      value: "/test_recursion_320"
+      attributes:
+      - check_all: 'yes'
+      - realtime: 'yes'
+      - recursion_level: "320"
+  - directories:
+      value: "/test recursion 320"
+      attributes:
+      - check_all: 'yes'
+      - realtime: 'yes'
+      - recursion_level: "320"
+
+# conf 3
+- tags:
+  - whodata
+  apply_to_modules:
+  - test_recursion_level
+  section: syscheck
+  elements:
+  - disabled:
+      value: 'no'
+  - directories:
+      value: "/test_no_recursion"
+      attributes:
+      - check_all: 'yes'
+      - whodata: 'yes'
+      - recursion_level: "2"
+  - directories:
+      value: "/test no recursion"
+      attributes:
+      - check_all: 'yes'
+      - whodata: 'yes'
+      - recursion_level: "2"
+  - directories:
+      value: "/test_recursion_1"
+      attributes:
+      - check_all: 'yes'
+      - whodata: 'yes'
+      - recursion_level: "3"
+  - directories:
+      value: "/test recursion 1"
+      attributes:
+      - check_all: 'yes'
+      - whodata: 'yes'
+      - recursion_level: "3"
+  - directories:
+      value: "/test_recursion_5"
+      attributes:
+      - check_all: 'yes'
+      - whodata: 'yes'
+      - recursion_level: "7"
+  - directories:
+      value: "/test recursion 5"
+      attributes:
+      - check_all: 'yes'
+      - whodata: 'yes'
+      - recursion_level: "7"
+  - directories:
+      value: "/test_recursion_320"
+      attributes:
+      - check_all: 'yes'
+      - whodata: 'yes'
+      - recursion_level: "320"
+  - directories:
+      value: "/test recursion 320"
+      attributes:
+      - check_all: 'yes'
+      - whodata: 'yes'
+      - recursion_level: "320"
```

Comment:
Maybe the configuration could be defined once and be parametrized using the `params` and `metadata` parameters of `load_wazuh_configurations` function.

Reply:
The config file has been refactorized to use `params` and `metadata` as suggested. The necessary modifications have been made to the test function to correctly use this new configuration file.